python/README.md

```diff
@@ -54,10 +54,12 @@ Setting `maxParallelForks` isn't required anymore since now Gradle parallelism i
 
 ### Test stats
 
-Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=23&leftFill=) (**1312**/5787 passed)
+Current status: ![coverage](http://www.yarntomato.com/percentbarmaker/button.php?barPosition=23&leftFill=) (**1314**/5787 passed)
 
 ### History (newest on top)
 
+* after supporting short, byte and float: **1314**/5787 passed (+2)
```

Comment:
Minor: strictly speaking, judging by `failed-tests.txt`, only tests for floats are affected. And this can't be named as full support of this stuff, just constants. For integers, there are much more changes to name it "initial support", and for floats, I guess some additional handing will be needed too as they have 32 bits, not Python's 64 :)

Although, JS has only 64 bit floats too, so maybe we can easily reuse the logic of emulating 32 bit floats.

```suggestion
* after supporting float constants: **1314**/5787 passed (+2)
```

Reply:
Yeah, I was surprised by the low number of new box tests passing. As I wrote in the commit message, the goal was to just somehow translate the IR entities, and not support it fully :)